app/scripts/modules/oracle/serverGroup/details/serverGroupDetails.controller.js

```diff
@@ -0,0 +1,224 @@
+'use strict';
+
+let angular = require('angular');
+
+import {CONFIRMATION_MODAL_SERVICE} from 'core/confirmationModal/confirmationModal.service';
+import {SERVER_GROUP_READER} from 'core/serverGroup/serverGroupReader.service';
+import {SERVER_GROUP_WRITER} from 'core/serverGroup/serverGroupWriter.service';
+import {SERVER_GROUP_WARNING_MESSAGE_SERVICE} from 'core/serverGroup/details/serverGroupWarningMessage.service';
+import {NETWORK_READ_SERVICE} from 'core/network/network.read.service';
+import {SUBNET_READ_SERVICE} from 'core/subnet/subnet.read.service';
+
+module.exports = angular.module('spinnaker.oraclebmcs.serverGroup.details.controller', [
+  require('angular-ui-router').default,
+  SERVER_GROUP_READER,
+  CONFIRMATION_MODAL_SERVICE,
+  SERVER_GROUP_WRITER,
+  SERVER_GROUP_WARNING_MESSAGE_SERVICE,
+  NETWORK_READ_SERVICE,
+  SUBNET_READ_SERVICE,
+  require('../../image/image.reader.js'),
+  require('./resize/resizeServerGroup.controller.js'),
+  require('./rollback/rollbackServerGroup.controller.js'),
+])
+  .controller('oraclebmcsServerGroupDetailsCtrl', function ($scope,
+                                                            $state,
+                                                            $uibModal,
+                                                            app,
+                                                            serverGroup,
+                                                            confirmationModalService,
+                                                            serverGroupReader,
+                                                            serverGroupWriter,
+                                                            networkReader,
+                                                            subnetReader,
+                                                            oraclebmcsImageReader,
+                                                            serverGroupWarningMessageService) {
+
+    const provider = 'oraclebmcs';
+
+    this.application = app;
+    this.serverGroup = serverGroup;
+
+    this.state = {
+      loading: true
+    };
+
+    /////////////////////////////////////////////////////////
+    // Fetch data
+    /////////////////////////////////////////////////////////
+
+    let retrieveServerGroup = () => {
+      return serverGroupReader
+        .getServerGroup(app.name, serverGroup.accountId, serverGroup.region, serverGroup.name)
+        .then((details) => {
+          cancelLoader();
```

Comment:
Will re-think this section in a subsequent PR. Works fine as is, but should really call this once all the other elements have loaded.